src/Root.tsx

```diff
@@ -0,0 +1,22 @@
+import * as Router from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage';
+import { App } from './App';
+
+export const Root = () => {
+  return (
+    <Router.HashRouter>
```

Comment:
Consider using destructuring 
```suggestion
    <HashRouter>
```